src/components/camera.js

```diff
@@ -87,6 +88,30 @@ module.exports.Component = registerComponent('camera', {
     });
   },
 
+  /**
+   * Enables updating camera.userHeight
+   */
+  updateHeightOffset: function () {
+    var currentPosition;
+    var headsetConnected;
+    var el = this.el;
+    var oldHeightOffset = this.userHeightOffset;
```

Comment:
I think no since I can't test headset at the moment. and to remove `userHeightOffset` the logic of  `enter-vr` and  `exit-vr` have to be modified a bit.